bolts/data/datasets/vision/celeba.py

```diff
@@ -149,11 +148,11 @@ def __init__(
         )
 
         x = np.array(attrs.index)
-        s = torch.as_tensor(attrs[subclass.name].to_numpy())
-        y = torch.as_tensor(attrs[superclass.name].to_numpy())
+        _s = torch.as_tensor(attrs[subclass.name].to_numpy())
+        _y = torch.as_tensor(attrs[superclass.name].to_numpy())
```

Comment:
Why?

Reply:
because then we would have easily spotted that `y` wasn't used

Reply:
Okay, fair enough, but maybe more meaningful pairs of names? Something like s_rad/y_rad and s_bern/y_bern.